### ISSUE TITLE
√Letter - Root Letter - (543240)

### ISSUE THREAD

**Issue by sat3ll** [ValveSoftware/Proton#4990]
# Whitelist Request
- Name of the game to be whitelisted: [√Letter - Root Letter -](https://store.steampowered.com/app/543240)
- Steam AppID of the game: 543240

## System Information
- GPU: Radeon VII
- Driver/LLVM version:  Mesa 21.1.4, LLVM 11.1.0
- Distro version: Fedora 34 with Freedesktop.org 20.08.14 (Flatpak runtime)
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/sat3ll/63c0cc18cc0231616936dca6818a66bd
- Proton version: _Proton 6.3-5_

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [ ] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [x] I haven't experienced any issues.
- [ ] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are remaining issues:

## Notes
Finished the game twice, everything is working flawlessly.

**Comment by robotjunkyard** [comment]
No, not everything works flawlessly -- video-based cutscenes do not play properly, instead showing a "test pattern" for a few seconds every time a fullscreen video is supposed to play  (Proton 9.0-3)
